test/models/ProcessContextSpec.scala

```diff
@@ -23,7 +23,7 @@ import play.api.inject.Injector
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import services.SecuredProcessBuilder
 
-class ProcessContextSpec extends BaseSpec with GuiceOneAppPerSuite {
+class GuidanceSessionSpec extends BaseSpec with GuiceOneAppPerSuite {
```

Comment:
Would we not rename this file instead of leaving it as ProcessContextSpec' ?

Reply:
renamed and pushed to branch

Reply:
Resolved